tests/scheduled_checks.py

```diff
@@ -0,0 +1,51 @@
+"""
+These tests are not called on every push or pull request. They're time-intensive, so on GitHub they're only called
+monthly. To run them locally, use the following command:
+
+    pytest -s -rpP tests/scheduled_checks.py
+
+Because the filename doesn't begin with test_, it's not automatically run with pytest. The -s flag is used to print
+the results of the tests, and the -rpP flags are used to suppress the output of the tests that pass.
+
+"""
+from astroquery.simbad import Simbad
+
+from astrodbkit2.utils import _name_formatter
+
+def test_SIMBAD_resolvable(db):
+    # Verify that sources have SIMBAD-resolvable names```
+    results = db.query(db.Sources.c.source).all()
+    name_list = [s[0] for s in results]
+
+    # Include all SIMBAD Identifiers and our provided name to the SIMBAD query results table.
+    Simbad.add_votable_fields("ids")
+    Simbad.add_votable_fields("typed_id")
+
+    simbad_results = Simbad.query_objects(name_list)
+    not_found_in_simbad = []
+    multiple_matches_in_db = []
```

Comment:
Are these variables used? They sound similar to `no_result_rows` and `duplicate_rows` below which are the ones that are actually populated.

Reply:
Tried running in CodeSpaces and it failed since `no_result_rows` and `duplicate_rows` are undefined. You need to rename one or other other